src/main/java/core/basesyntax/Figure.java

```diff
@@ -0,0 +1,18 @@
+package core.basesyntax;
+
+public class Figure implements CalculateArea, Drawable {
+    private int height;
```

Comment:
![image](https://user-images.githubusercontent.com/11480043/227712611-9e81df91-4e07-43e6-bda7-f95ae6e8ca97.png)
fix according to task description

Reply:
not fixed